src/fileimpl.cpp

```diff
@@ -188,6 +188,7 @@ class Grouping
       clusterCache(envValue("ZIM_CLUSTERCACHE", CLUSTER_CACHE_SIZE)),
       m_newNamespaceScheme(false),
       m_hasFrontArticlesIndex(true),
+      m_hasExhaustiveEntryTitleListing(true),
```

Comment:
This change is superseded by a later commit. It would be nice if the PR history contains only the final version.